app/viewmodels/checkAnswers/euDetails/VatRegisteredSummary.scala

```diff
@@ -26,7 +26,13 @@ import viewmodels.implicits._
 
 object VatRegisteredSummary {
 
-  def row(answers: UserAnswers, index: Index)(implicit messages: Messages): Option[SummaryListRow] =
+  def row(answers: UserAnswers, index: Index, currentMode: Mode)(implicit messages: Messages): Option[SummaryListRow] = {
+
+    val changeLinkMode = currentMode match {
+      case NormalMode    => CheckLoopMode
+      case CheckMode     => CheckMode
```

Comment:
As per previous comment

Reply:
No, and I'll add a case which throws an exception if that does occur